test/unit/header.test.js

```diff
@@ -229,4 +238,13 @@ describe('Header', function () {
             expect(Header.isHeader()).to.be.false;
         });
     });
+
+    describe('toString', function () {
+        it('should convert the header to a single header string', function () {
+            // eslint-disable-next-line max-len
+            var rawHeader = 'Content-Type: application/json\nAuthorization: Hawk id="dh37fgj492je", ts="1448549987", nonce="eOJZCd", mac="O2TFlvAlMvKVSKOzc6XkfU6+5285k5p3m5dAjxumo2k="',
+                newHeader = new Header(rawHeader);
+            expect(newHeader.toString()).equals(rawHeader);
+        });
+    });
```

Comment:
Let's also add a test where we create a new header by passing an object to `new Header()` and check the `toString` output.